rom/dev/src/flow/debug_unlock.rs

```diff
@@ -121,12 +121,12 @@ fn handle_manufacturing(env: &mut RomEnv) -> CaliptraResult<()> {
     result
 }
 
-fn payload_length(length: [u8; 3]) -> usize {
-    let mut len: usize = 0;
-    len |= length[0] as usize;
-    len |= (length[1] as usize) << 8;
-    len |= (length[2] as usize) << 16;
-    len * size_of::<u32>()
+fn bytes_to_usize(input: [u8; 3]) -> usize {
```

Comment:
If we're pulling it out of context, it might be good to give a quick comment explaining the input and output
```suggestion
/// Converts a little-endian 3-byte count of number of dwords to the corresponding length in bytes.
fn bytes_to_usize(input: [u8; 3]) -> usize {
```

Reply:
Done, thanks.